draft-yasskin-http-origin-signed-responses.md

```diff
@@ -1342,24 +1307,36 @@ signed exchange.
 
 Specification: This document
 
-## Internet Media Type application/http-exchange+cbor
+## Internet Media Type application/signed-exchange
 
 Type name:  application
 
-Subtype name:  http-exchange+cbor
+Subtype name:  signed-exchange
 
-Required parameters:  N/A
+Required parameters:
+
+* v: A string denoting the version of the file format. ({{!RFC5234}} ABNF:
+  `version = DIGIT/%x61-7A`) The version defined in this specification is `1`.
+  When used with the `Accept` header field (Section 5.3.1 of {{!RFC7231}}), this
+  parameter can be a comma (,)-separated list of version strings. ({{!RFC5234}}
+  ABNF: `version-list = version *( "," version )`) The server is then expected
+  to reply with a resource using a particular version from that list.
+
+  Note: RFC EDITOR PLEASE DELETE THIS NOTE; Implementations of drafts of this
+  specification MUST NOT use simple integers to describe their versions, and
+  MUST instead define implementation-specific strings to identify which draft is
+  implemented.
```

Comment:
Nit: That then suggests that versions aren't digits but opaque strings, right?

Reply:
Yep, I believe I included that change in b95e497dd2bfb780504220ccd4ebe85fe64baf56 (especially `v: A string denoting the version`), but if you see a word I missed, let me know.